.eslintrc.yml

```diff
@@ -4,6 +4,8 @@ extends:
   - plugin:import/errors
   - plugin:import/warnings
   - plugin:node/recommended
+rules:
+  node/shebang: off
```

Comment:
Was the missing shebang in `dev/chokidar.js` an issue? Even with `yarn capture:manual`?

Reply:
There are two ways to execute a file: `node dev/chokidar.js` and `dev/chokidar.js`. The second one needs two things: the file must be executable and has a shebang. The first one doesn't care about those two things.

`dev/chokidar.js` was executable but didn't have a shebang. This doesn't look logical to my eyes. I choose to add the shebang, but removing the executable permissions is also an option (and I have already modified the eslint rule for `test/generate_property_json.js`).

`yarn capture:manual` executes it with `node dev/chokidar.js`, so there is no issue here.